### ISSUE TITLE
App don't even start in linux

### ISSUE THREAD

**Issue by gerordu** [ytmdesktop/ytmdesktop#1097]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Opening application keeps app loading indefinitely (a red spin in the middle of black screen is showed only), when I ran with console I can see this error:

> [!] To add translation for your introduced feature, you may navigate to 
[!] /home/<USER>/.config/youtube-music-desktop-app/locales to modify the your locale file.
[!] Then kindly open a PR to ytmdesktop-locales repo. :)
Checking for update
(node:13803) UnhandledPromiseRejectionWarning: Error: ERR_FAILED (-2) loading 'https://music.youtube.com'
    at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
    at Object.stopLoadingListener (electron/js2c/browser_init.js:205:1868)
    at Object.emit (events.js:327:22)
(node:13803) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:13803) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Update for version 1.14.2 is not available (latest version: 1.13.0, downgrade is disallowed).

I donno why, and I've not tried this on another device yet.

### Expected Behavior

App running normally, showing YouTube Music dashboard.

### Steps To Reproduce

1. Download YouTube Music Desktop App appimage(1.14.2 ver)
2. Right click > properties > Run as program
3. Run app

### YTMDesktop

1.14.2

### OS

Fedora

### OS Version

37

### Arch

x64

### Installation way

AppImage

**Comment by trigger337** [comment]
I can confirm this issue for versions 1.13.0 and 1.14.2 on Fedora 38 when running AppImage. Flatpak works just fine.
1.14.2:
> Checking for update
> (node:69959) UnhandledPromiseRejectionWarning: Error: ERR_FAILED (-2) loading 'https://music.youtube.com'
>     at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
>     at Object.stopLoadingListener (electron/js2c/browser_init.js:205:1868)
>     at Object.emit (events.js:327:22)
> (node:69959) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
> (node:69959) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> Update for version 1.14.2 is not available (latest version: 1.13.0, downgrade is disallowed).

1.13.0:

> Checking for update
> (node:70448) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
> Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).

1.13.0 also segfaults if I close it in this state

**Comment by trigger337** [comment]
Decided to test if it has problems with sandbox like BetterDiscord and running with sandbox disabled fixes the problem `./youtube-music-desktop-app --no-sandbox`
Any ideas?

**Comment by nealbailey** [comment]
Any word on when the AppImage will work again in Linux? 
I'm using using Pop!_OS 22.04 LTS x86_64 (kernel 6.2.6-76060206-generic, node v14.21.3) the appImage dies on startup 

> Checking for update
> (node:8900) UnhandledPromiseRejectionWarning: Error: ERR_FAILED (-2) loading 'https://music.youtube.com'
>     at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
>     at Object.stopLoadingListener (electron/js2c/browser_init.js:205:1868)
>     at Object.emit (events.js:327:22)
> (node:8900) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
> (node:8900) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> Update for version 1.14.2 is not available (latest version: 1.13.0, downgrade is disallowed).
> 

The version of the app in the Pop Shop works (1.13.0) but you can't sign into your premium account so its pretty much useless.

**Comment by zunino** [comment]
I'm getting the same behavior, i.e. app launches, its window is displayed with no content other than the spinning animation, which runs indefinitely. The console displays only a `DeprecationWarning` about a `Buffer()` call. When I close the window, a "Segmentation fault" is displayed and the process terminates.

OS: Debian 12 Bookworm
YTMD: YouTube-Music-Desktop-App-1.13.0.AppImage

**Comment by sjfloat** [comment]
I'm getting the same behavior. I can add that running it with `strace` I can see that it's hanging up at:

```
poll([{fd=3, events=POLLIN}, {fd=4, events=POLLIN}], 2, -1Checking for update
```

which is followed by the following from the app itself to stderr:


```
(node:2) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

and to stdout:

```
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).
```

**Comment by Alipoodle** [comment]
Closing issue due to no longer supporting YouTube Music Desktop v1 + Release of YouTube Music v2.
Please re-open a NEW issue if this is still occurring in the latest build of the application, however as a re-write has been done on the entire project most issues will be fixed, no longer a feature or re-worked.
